diff -Naur --exclude '*.orig' db-4.7.25/dist/configure.ac db-4.7.25-jni-include-dir/dist/configure.ac
--- db-4.7.25/dist/configure.ac	2008-05-06 05:03:36.000000000 +0200
+++ db-4.7.25-jni-include-dir/dist/configure.ac	2009-01-04 21:52:42.000000000 +0100
@@ -418,7 +418,7 @@
 	AC_PROG_JAVAC
 	AC_PROG_JAR
 	AC_PROG_JAVA
-	AC_JNI_INCLUDE_DIR
+	JNI_INCLUDE_DIRS="/usr/lib/jvm/java-1.5.0-gcj/include"
 
 	AC_MSG_CHECKING(java version)
         case "$JAVA" in
